djangosaml2/views.py

```diff
@@ -533,6 +538,15 @@ def get(self, request, *args, **kwargs):
             'Could not logout because there only the HTTP_REDIRECT is supported')
         return HttpResponseServerError('Logout Binding not supported')
 
+    def custom_error_handler(self, request, exception, *args, **kwargs):
+        """ Subclasses may override this method to implement custom logic for
+            handling logout errors.
+
+            For example, a site may want to perform additional logic and redirect
+            users somewhere other than the LOGOUT_REDIRECT_URL.
+        """
+        return None
```

Comment:
I think we can default to returning `return HttpResponseRedirect(getattr(settings, 'LOGOUT_REDIRECT_URL', '/'))` and just always call this method.

Reply:
Done. Not sure if you have any preferences with the method name. I renamed it to `customizable_error_handler()` to fit in with the custom methods on the ACS service for now, but I'm used to having method names as verbs.

Reply:
Hmm I was thinking something more descriptive like `handle_unsupported_slo_exception`?

Reply:
Yeah that sounds good